mesonbuild/build.py

```diff
@@ -537,13 +538,14 @@ def __post_init__(self, overrides: T.Optional[T.Dict[OptionKey, str]]) -> None:
         else:
             ovr = {}
         self.options = OptionOverrideProxy(ovr, self.environment.coredata.options, self.subproject)
-        # XXX: this should happen in the interpreter
+        self.install_name_auto = None
         if has_path_sep(self.name):
-            # Fix failing test 53 when this becomes an error.
             mlog.warning(textwrap.dedent(f'''\
                 Target "{self.name}" has a path separator in its name.
                 This is not supported, it can cause unexpected failures and will become
-                a hard error in the future.'''))
+                a hard error in the future. Consider using the install-name property instead'''))
```

Comment:
`install-name` -> `install_name`?

Reply:
yup. not sure where I got the - name from ...